asdf_astropy/resources/manifests/astropy-1.1.0.yaml

```diff
@@ -54,3 +54,20 @@ tags:
   title: NdarrayMixin column.
   description: |-
     Represents an astropy.table.NdarrayMixin instance.
+- tag_uri: tag:astropy.org:astropy/slicedwcs/slicedwcs-1.0.0
+  schema_uri: http://astropy.org/schemas/astropy/slicedwcs/slicedwcs-1.0.0
+  title: Represents an instance of SlicedLowLevelWCS
+  description: |-
```

Comment:
Would you make the schema title/descriptions match the ones here?

Reply:
Sure